test/test_utils/__init__.py

```diff
@@ -248,6 +254,13 @@ def get_dockerfile_path_for_image(image_uri):
         if "example" not in path
     ]
 
+    if not jdk_version and any("jdk" in path for path in dockerfiles_list):
+        raise LookupError(
```

Comment:
this needs to be updated - jdk version should not be in the image URI - does dockerfiles_list not pick it up? it should as it should look at subdirs